src/run_validation.py

```diff
@@ -21,8 +21,16 @@ def main() -> int:
     logging.getLogger("urllib3").setLevel(logging.WARNING)
 
     with TemporaryDirectory() as work_dir:
-        test_result = ValidationTestRunner.dispatch(args, args.distribution, work_dir).run()
-        logging.info(f'final test_result = {test_result}')
+        if args.distribution == "docker":
+            docker_source = args.docker_source
+            for source in docker_source:
+                docker_args = args
+                docker_args.docker_source = source
+                test_result = ValidationTestRunner.dispatch(docker_args, args.distribution, work_dir).run()
+
+        else:
+            test_result = ValidationTestRunner.dispatch(args, args.distribution, work_dir).run()
```

Comment:
Why this change?

Reply:
It calls docker validation independently based on the values in --docker-source.

Reply:
I see. Now you can call multiple of that.
But wont that be an issue if a previous dispatch already occupy same ports?
I assume you are not binding ports to the system right?

Reply:
No, tested the changes, working as expected. The second pull takes place only after the cleanup. How ever it's mostly used by the release-promotion job.

Reply:
After offline discussion, dispatch is not async but rather wait for completion.

Thanks.